.stylelintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: '@yoo-digital/stylelint-config-sass',
+  ignoreFiles: [
```

Comment:
I think this is necessary, otherwise the stylelint would want to adjust the minified css in the dist folder. With this setting it does not do that. Including it in the .gitignore, does not resolve this issue.